lib/library.go

```diff
@@ -334,6 +334,74 @@ func RecoverAccount(password, mnemonic *C.char) *C.char {
 	return C.CString(string(outBytes))
 }
 
+// StartOnboarding initialize the onboarding with n random accounts
+//export StartOnboarding
+func StartOnboarding(n, mnemonicPhraseLength C.int) *C.char {
+	accounts, err := statusBackend.AccountManager().StartOnboarding(int(n), int(mnemonicPhraseLength))
+
+	errString := ""
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		errString = err.Error()
+	}
+
+	out := struct {
```

Comment:
In my opinion it's a bit confusing. I would declare `out` at the top of the function and if an error is detected, I would do `out.Error = errString`.

Finally, it would be something like this:
```go
out := struct {
    // ...
}

if err != nil {
    fmt.Fprintln(os.Stderr, err)
    out.Error = errString
} else {
    for _, account := range accounts {
      // ...
    }
}

outBytes, _ := json.Marshal(out)
return C.CString(string(outBytes))
```

Reply:
thank you @adambabik , it makes sense, I updated the code